### PR TITLE
[RM Ch 07] Cloud Infrastrucure Regulatory Compliance 

### PR DESCRIPTION
**Pull request by walterkozlowski**
Section dealing with regulatory frameworks: role in general and examples form few jurisdictions, to be extended in future releases.

### CODE REVIEW

File: doc/ref_model/chapters/chapter07.rst [hunk 1]
```diff
@@ -801,6 +801,124 @@ Security certification should encompass the following elements:
...
+and their partners, and ultimately to the end consumers of the telecommunication services. However, it also brings compliance challenges that can seem overwhelming. 
+The telecommunication industry players can reduce this overwhelm by arming themselves with information about which laws they need to comply with, why, and how.  
+
+The costs of non-compliance can be very serious. Organisations may not only have to contend with hefty fines and possible lawsuits, but they may also even end up 
+losing their reputation and eventually losing customers, with an obvious adverse impact on revenues and profitability. 
+
+Compliance means that an operator’s systems, processes, and workflows align with the requirements mandated by the regulatory regimes imposed by the relevant 
+governmental and industry regulatory bodies. The need for compliance extends to the cloud, so operators must ensure that any data transferred to and out, and stored 
+in their cloud infrastructure complies with all relevant data protection and privacy laws.
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter07.rst:816]: Suggest add of "location"
```suggestion
in their cloud infrastructure complies with all relevant data protection, including data residency, and privacy laws.
```

**Review by pgoyal01 — APPROVED** [review]
subject to a minor change

File: doc/ref_model/chapters/chapter07.rst (hunk 1 above)
Comment (jbhartley) [reply doc/ref_model/chapters/chapter07.rst:816]: @pgoyal01 @walterkozlowski typically when discussing data location (residency) in cloud industry this is covered as "data sovereignty".

**Review by jbhartley — APPROVED** [review]
Good quality and informative, particularly in listing the various frequently used standards and country regulatory bodies.

**Review by karinesevilla — APPROVED** [review]

**Pull request merged**